src/mono/mono/metadata/wrapper-types.h

```diff
@@ -20,6 +20,4 @@ WRAPPER(STELEMREF, "stelemref")
 WRAPPER(UNBOX, "unbox")
 WRAPPER(WRITE_BARRIER, "write-barrier")
 WRAPPER(OTHER, "other")
-WRAPPER(COMINTEROP_INVOKE, "cominterop-invoke")
-WRAPPER(COMINTEROP, "cominterop")
```

Comment:
@vargaz Do the wrapper types this leak out into public APIs anywhere? Do we need to keep the numbering the same?  ie, should these be replaced by

```c
WRAPPER(UNUSED1, "unused1")
WRAPPER(UNUSED2, "unused2")
```

Reply:
I looked through the public API and there's no way to get at the wrapper type from the embedding API surface.